diff --git a/CMakeLists.txt b/CMakeLists.txt
index f71e06631..b29d91c23 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -86,7 +86,7 @@ if(${CMAKE_SYSTEM_NAME} MATCHES "Android")
 endif()
 
 # We only support Vulkan on Unix, macOS (by MoltenVK), Android and Windows.
-if(ANDROID OR WIN32 OR (UNIX AND NOT ARM_NO_VULKAN))
+if(ANDROID OR WIN32 OR (UNIX AND NOT ARM_NO_VULKAN) OR FORCE_VULKAN)
 	set(VULKAN ON)
 endif()
 
